doc/src/site/sphinx/tutorials/sw_kmeans.rst

```diff
@@ -0,0 +1,121 @@
+Train KMeans Model in Sparkling Water
+--------------------------------------
+
+Sparkling Water provides API for H2O KMeans in Scala and Python. All available parameters
+of H2OKmeans model are described at `H2O KMeans Parameters`_.
```

Comment:
Do we need `_` there?

Reply:
Yup, this represents internal link. After rendering, it will create a hyperlink to the section bellow

Reply:
good to know, thanks a lot!